samples/face_reid/index_builder.yml

```diff
@@ -0,0 +1,92 @@
+name: face_reid_index_builder
+
+parameters:
+  frame:
+    width: 1280
+    height: 720
+
+  face_width: 112
+  face_height: 112
+  detection_model_name: yolov5nface
+
+  reid_model_name: adaface
+
+pipeline:
+
+  elements:
+    # detector
+    - element: nvinfer@complex_model
+      name: ${parameters.detection_model_name}
+      model:
+        remote:
+          url: s3://savant-data/models/yolov5face/${parameters.detection_model_name}.zip
+          checksum_url: s3://savant-data/models/yolov5face/${parameters.detection_model_name}.md5
+          parameters:
+            endpoint: https://eu-central-1.linodeobjects.com
+        format: onnx
+        config_file: yolov5n-face.txt
+        input:
+          shape: [3, 640, 640]
+        output:
+          layer_names: [ 'output' ]
+          converter:
+            module: savant.converter.yolo_v5face
+            class_name: YoloV5faceConverter
+            kwargs:
+              confidence_threshold: 0.6
+              nms_iou_threshold: 0.5
+          objects:
+            - class_id: 0
+              label: face
+              selector:
+                module: samples.face_reid.selector
+                class_name: MinMaxSizeBBoxSelector
+                kwargs:
+                  min_width: 40
+                  min_height: 40
+          attributes:
+            - name: landmarks
+
+    - element: nvinfer@attribute_model
+      name: ${parameters.reid_model_name}
+      model:
+        remote:
+          url: s3://savant-data/models/${parameters.reid_model_name}/${parameters.reid_model_name}_ir50_webface4m_90fb74c.zip
```

Comment:
There is no such path s3://savant-data/models/adaface. I think file module.yml contains the correct reid_model_name (suffix included).